.changeset/sixty-scissors-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pnpm/plugin-commands-installation": patch
```

Comment:
Possible this should just be `major`.

https://github.com/pnpm/pnpm/pull/6403/files#r1167671430

Reply:
ok, you can do a major

Reply:
Will do! Latest push switches this to `major`.